ex03.ex

```diff
@@ -55,7 +55,32 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  
+#   Another Solution using Guard
+#   def odd_even([]), do: []
+#   def odd_even([head | tail]) when rem(head, 2) == 0 do
+#       [:even | odd_even(tail)]
+#   end
+#   def odd_even([_head|tail]) do
+#       [:odd | odd_even(tail)]
+#   end
+
+  
+  # Another Solution using if
+  # def odd_even([]), do: []
+  # def odd_even([head | tail]) do
+  #   if Integer.is_even(head), do: [:even|odd_even(tail)], else: [:odd|odd_even(tail)]
+  # end
+
+
+  #Solution using cond
```

Comment:
I'm not deducting anything here, but I'd encourage you always to look for a solution that avoids `cond` or `if`.  You'll find it helps keep all the code in your functions at the same level. For example

~~~ elixir
  def odd_even([]), do: []
  def odd_even([ h | t ]), do: [ atom_for(h) | odd_even(t) ]

  defp atom_for(n) when Integer.is_odd(n),  do:  :odd
  defp atom_for(n) when Integer.is_even(n), do:  :even
~~~